spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/sql/CassandraDataSourceSpec.scala

```diff
@@ -52,14 +52,9 @@ class CassandraDataSourceSpec extends SparkCassandraITFlatSpecBase {
         |CREATE TEMPORARY TABLE $tmpTable
         |USING org.apache.spark.sql.cassandra
         |OPTIONS (
-        | c_table "$table",
+        | table "$table",
         | keyspace "$keyspace",
-        | push_down "$pushDown",
-        | spark_cassandra_input_page_row_size "10",
```

Comment:
Huh, so the brutal truth is that they *don't work* now and we can't do anything about it. At least not with Spark 1.4.0. Instead of adding support for "." they removed support for "_" and now we have no word separator at all.